cryptofeed/exchanges/bybit.py

```diff
@@ -446,12 +453,10 @@ async def _order(self, msg: dict, timestamp: float):
                 'create_type': data.get('create_type'),
             }
             LOG.debug(f"Handle order msg: {data_to_log}")
-            await self.callback(ORDER_INFO, feed=self.id, symbol=symbol, data=data, receipt_timestamp=timestamp)
+            await self.callback(ORDER_INFO, feed=self.id, receipt_timestamp=timestamp, **data)
 
     def parse_balance(self, data: dict[str, Any]) -> dict[str, Any]:
-        balance: dict[str, Any] = {
-            'timestamp': self.timestamp_normalize(data['timestamp'])
```

Comment:
Don't we need timestamp for balance data ?

Reply:
Look at line 487.